tests/kernel/sched/schedule_api/prj_multiq.conf

```diff
@@ -5,3 +5,5 @@ CONFIG_SCHED_MULTIQ=y
 CONFIG_MAX_THREAD_BYTES=5
 CONFIG_MP_MAX_NUM_CPUS=1
 CONFIG_ZTEST_FATAL_HOOK=y
+CONFIG_NUM_COOP_PRIORITIES=20
+CONFIG_NUM_PREEMPT_PRIORITIES=20
```

Comment:
Is this mainly just to check that this will work when there are > 32 total priorities?

It might not be necessary to explicitly do that, since the `BIT_*()` macros should already be tested to work in those scenarios (i.e. there may be coverage already)

Reply:
> Is this mainly just to check that this will work when there are > 32 total priorities?
> 
Yes, it is.
> It might not be necessary to explicitly do that, since the `BIT_*()` macros should already be tested to work in those scenarios (i.e. there may be coverage already)

Ok. Thanks for your information.

Reply:
Hi @cfriedt,
I have a question which is how can I check this feature be tested?